eth/tracers/api_blocktrace.go

```diff
@@ -391,15 +391,13 @@ func (api *API) fillBlockTrace(env *traceEnv, block *types.Block) (*types.BlockT
 	}
 
 	// only zktrie model has the ability to get `mptwitness`.
-	if api.backend.ChainConfig().Zktrie {
+	if api.backend.ChainConfig().Scroll.ZktrieEnabled() {
 		if err := zkproof.FillBlockTraceForMPTWitness(zkproof.MPTWitnessType(api.backend.CacheConfig().MPTWitness), blockTrace); err != nil {
 			log.Error("fill mpt witness fail", "error", err)
 		}
 	}
 
-	if api.backend.ChainConfig().UsingScroll {
-		blockTrace.WithdrawTrieRoot = withdrawtrie.ReadWTRSlot(rcfg.L2MessageQueueAddress, env.state)
-	}
+	blockTrace.WithdrawTrieRoot = withdrawtrie.ReadWTRSlot(rcfg.L2MessageQueueAddress, env.state)
```

Comment:
This is not controlled by any config now. Is it compatible with other geth configs?